client/lib/plans/index.js

```diff
@@ -220,19 +220,30 @@ export function plansLink( url, site, intervalType ) {
 
 export function applyTestFiltersToPlansList( planName ) {
 	const filteredPlanConstantObj = { ...plansList[ planName ] };
-	let filteredPlanFeaturesConstantObj = pick( featuresList, plansList[ planName ].getFeatures() );
+	let filteredPlanFeaturesConstantList = plansList[ planName ].getFeatures();
 
 	const removeDisabledFeatures = () => {
 		if ( ! isWordadsInstantActivationEnabled() ) {
-			filteredPlanFeaturesConstantObj = pickBy( filteredPlanFeaturesConstantObj,
-				( value, key ) => key !== FEATURE_WORDADS_INSTANT
+			filteredPlanFeaturesConstantList = filter( filteredPlanFeaturesConstantList,
+				( value ) => value !== FEATURE_WORDADS_INSTANT
 			);
 		}
+
 		if ( ! isGoogleVouchersEnabled() ) {
-			filteredPlanFeaturesConstantObj = pickBy( filteredPlanFeaturesConstantObj,
-				( value, key ) => key !== FEATURE_GOOGLE_AD_CREDITS
+			filteredPlanFeaturesConstantList = filter( filteredPlanFeaturesConstantList,
+				( value ) => value !== FEATURE_GOOGLE_AD_VOUCHERS_100 &&
+					value !== FEATURE_GOOGLE_WORDADS_AD_VOUCHERS_200
 			);
 		}
+
+		if ( ! isWordpressAdCreditsEnabled() ) {
+			const wordpressAdCreditsFeatureIndex = filteredPlanFeaturesConstantList.
+				indexOf( FEATURE_GOOGLE_WORDADS_AD_VOUCHERS_200 );
+
+			if ( wordpressAdCreditsFeatureIndex !== -1 ) {
+				filteredPlanFeaturesConstantList[ wordpressAdCreditsFeatureIndex ] = FEATURE_GOOGLE_AD_VOUCHERS_100;
```

Comment:
So, Premium plan has feature `FEATURE_GOOGLE_AD_VOUCHERS_100` and Business plan `FEATURE_GOOGLE_WORDADS_AD_VOUCHERS_200`. They are on the same spot/order so users can compare.

Now, if we disable wordpressAdCredits abtest, we only give users the $100 Google Voucher in both the Premium and Business plans, right? If so, I need to replace the `FEATURE_GOOGLE_WORDADS_AD_VOUCHERS_200` feature of Business plan with `FEATURE_GOOGLE_AD_VOUCHERS_100` so it matches.

The whole point of this PR is that in @apeatling's latest designs for plans redesign, Advertising Vouchers have their values in titles, $100 Adv. vouchers for Premium and $200 Adv. vouchers for Business. Therefore, I had to create two separate features even though wordpressAdCredits is enabled only and only if googleVouchers is enabled too. However, you can have googleVouchers without wordpressAdCredits, that's why I need to do the above.